integration-tests/alfresco-enterprise-62/overload.gradle

```diff
@@ -1,5 +1,6 @@
 ext {
-    alfrescoBaseWar = 'org.alfresco:content-services:6.2.1@war'
-    alfrescoBaseImage = 'hub.xenit.eu/alfresco-enterprise/alfresco-repository-enterprise:6.2.1'
+    alfrescoBaseWarBom = 'org.alfresco:acs-packaging:6.2.2.19'
+    alfrescoBaseWar = 'org.alfresco:content-services@war'
+    alfrescoBaseImage = 'hub.xenit.eu/alfresco-enterprise/alfresco-repository-enterprise:6.2.2'
```

Comment:
Can we pin this base image to exactly `6.2.2.19`?

Else, integration tests will fail when docker-alfresco builds a new image for a future Alfresco version.

Reply:
Pinned